nshlib/nsh_altconsole.c

```diff
@@ -62,7 +62,7 @@ static int nsh_clone_console(FAR struct console_stdio_s *pstate)
 
   /* Flush stderr: we only flush stderr if we opened the alternative one */
 
-  fflush(stderr);
+  fsync(STDERR_FILENO);
```

Comment:
le's remove fsync call since fflush just flush the buffer at stdio layer, not vfs layer. So, we don't need do anything here after changing file stream to file handle.